dbos/cli.py

```diff
@@ -64,9 +72,117 @@ def signal_handler(signum: int, frame: Any) -> None:
         process.wait()
 
 
+def get_template_directory() -> str:
+    import dbos
+
+    package_dir = path.abspath(path.dirname(dbos.__file__))
+    return path.join(package_dir, "templates")
+
+
+def copy_dbos_config(src: str, dst: str, project_name: str) -> None:
+    package_name = project_name.replace("-", "_")
+    db_name = package_name if not package_name[0].isdigit() else f"_{package_name}"
+
+    with open(src, "r") as file:
+        content = file.read()
+
+    content = content.replace("${APP_NAME}", project_name)
+    content = content.replace("${APP_DB_NAME}", db_name)
+    content = content.replace("${APP_PACKAGE_NAME}", package_name)
+
+    try:
+        with open(dst, "x") as file:
+            file.write(content)
+    except FileExistsError:
```

Comment:
I think we should print a warning message to user that `dbos-config.yaml` already exists, skipping..